docs/conf.py

```diff
@@ -47,7 +47,6 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
     'sphinx.ext.viewcode',
-    'sphinx.ext.githubpages',
```

Comment:
This will break production of the cobbler docs publishing...

Reply:
No, but `sphinx.ext.githubpages` will break the build for CentOS7 due to an outdated version of Sphinx-docs. 
From @SchoolGuy:
>  I just looked at the github-pages-plugin of Sphinx and we don't need that because we don't publish the docs on GH-Pages. So we can remove this without any side effects in my eyes.